packages/api/src/pubsub.ts

```diff
@@ -7,10 +7,14 @@ export interface ChangesList {
 	change24h: number
 }
 
-const pubsub = new LittlePubSub()
+declare global {
+	var pubsub: LittlePubSub
+}
+
+globalThis.pubsub = globalThis.pubsub || new LittlePubSub()
```

Comment:
you are right, actually I don't like this. the methods in PubSub are static, you should not expose pubsub directly everywhere that is not good practice. The static class can be imported where code needs it and it prevents writing pubsub logic else where. Please revert that.

Reply:
I won't import the PubSub class to https://github.com/VandeurenGlenn/socket-request-server/blob/master/src/server.ts
But there yes have an option to pass pubsub, but everyone agrees globalThis.pubsub is globalThis.pubsub? :p

iow it is global already because using socket-request-server

Reply:
that's so weird lol... your lib should have an entry point, an `init` function or I don't know, exposing it to the whole architecture just for your lib to function...is not really a good paradigm right there... But I guess there is no other way around.